types/graphql-resolve-batch/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-namespace": false,
+        "array-type": [true, "array"]
```

Comment:
Why do you need this on?

Reply:
* I enabled namespaces because I wrote the tests that way. I'm happy to remove it and restructure the tests so they don't use namespaces, but this looked like a readable way to structure them :)
* array type is specificly set to array because dtslint would basically ping-pong between "using [] or Array<T>"